src/modules/stackoverflow/stackoverflow.api.js

```diff
@@ -0,0 +1,26 @@
+//import { setRepos, setError } from "./github.actions";
+
+class StackOverflowAPI {
+  constructor() {
+    this.url = "https://api.stackexchange.com/2.3/similar";
+  }
+  getData(topic) {
+    // https://api.stackexchange.com/2.3/similar?order=desc&sort=creation&title=javascript%20react&site=stackoverflow
+    console.log(
+      `${this.url}?order=desc&sort=creation&title=${topic}&site=stackoverflow`
+    );
+    return fetch(
+      `${this.url}?order=desc&sort=creation&title=${topic}&site=stackoverflow`
+    ).then((resp) => resp.json());
```

Comment:
Jeśli chodzi o "gotowe" sortowanie to po prostu można "poprosić" API o dane w odpowiedniej kolejności.
Wystarczy w funkcji zdefiniować odpowiedni parametr i go tutaj przekazać: `... sort=${sortWay} ...`